schedule_lessons/dashboard/urls.py

```diff
@@ -38,5 +38,6 @@
          name='clear_notifications'),
     path('block/<user_id>', views.block_user, name='block_user'),
     path('unblock/<user_id>', views.unblock_user, name='unblock_user'),
-    path('save_tutorial_preferences', views.save_tutorial_preferences, name='save_tutorial_preferences')
+    path('save_tutorial_preferences', views.save_tutorial_preferences, name='save_tutorial_preferences'),
+    path('settings/', views.settings, name='settings')
```

Comment:
why `settings/` instead of just `settings`?

Reply:
Good point. Actually, the urls above settings are inconsistent, they should have trailing slashes as well. I opened up issue #138, so we can fix this in another PR.